<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-02-21T06:33:17Z" name="Default suite" finished-at="2019-02-21T06:33:17Z" duration-ms="229">
    <groups>
    </groups>
    <test started-at="2019-02-21T06:33:17Z" name="Default test" finished-at="2019-02-21T06:33:17Z" duration-ms="229">
      <class name="Zipgo.InterCity.base.CreateTrip">
        <test-method signature="CreateTestTrip()[pri:1, instance:Zipgo.InterCity.base.CreateTrip@1649b0e6]" started-at="2019-02-21T12:03:17Z" name="CreateTestTrip" finished-at="2019-02-21T12:03:17Z" duration-ms="58" status="FAIL">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[./src/main/resources/CreateTrip.properties (No such file or directory)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: ./src/main/resources/CreateTrip.properties (No such file or directory)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:220)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:158)
	at Zipgo.InterCity.base.CreateTrip.CreateTestTrip(CreateTrip.java:110)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateTestTrip -->
        <test-method signature="sendMail()[pri:2, instance:Zipgo.InterCity.base.CreateTrip@1649b0e6]" started-at="2019-02-21T12:03:17Z" name="sendMail" finished-at="2019-02-21T12:03:17Z" duration-ms="121" status="FAIL">
          <exception class="org.apache.commons.mail.EmailException">
            <message>
              <![CDATA[Cannot attach file "./screenshot/2019-02-21SecondScreenShot.png"]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.apache.commons.mail.EmailException: Cannot attach file "./screenshot/2019-02-21SecondScreenShot.png"
	at org.apache.commons.mail.MultiPartEmail.attach(MultiPartEmail.java:333)
	at Zipgo.InterCity.base.CreateTrip.sendMail(CreateTrip.java:303)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.IOException: "./screenshot/2019-02-21SecondScreenShot.png" does not exist
	at org.apache.commons.mail.MultiPartEmail.attach(MultiPartEmail.java:322)
	... 26 more
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.commons.mail.EmailException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sendMail -->
      </class> <!-- Zipgo.InterCity.base.CreateTrip -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
